extras/logo/README.md

```diff
@@ -0,0 +1,20 @@
+# OpenSlides logo variants
```

Comment:
Can you please use ReStructeredText instead of MarkDown because we do so in other READMEs.